packages/ember-htmlbars/tests/helpers/each_test.js

```diff
@@ -276,6 +276,35 @@ QUnit.test("View should not use keyword incorrectly - Issue #1315", function() {
   equal(view.$().text(), 'X-1:One 2:Two Y-1:One 2:Two ');
 });
 
+Ember.runInDebug(function() {
```

Comment:
I do not believe that this is processed properly in tests.  You should use something like the following:

```javascript
// at the top
/* globals EmberDev */

// then in this location:
if (EmberDev && EmberDev.runningProdBuild) {

}
```